README.md

```diff
@@ -20,6 +20,53 @@ Modern vision of the mobile application for the Open edX platform from Raccoon G
+
+Before retrieving the translations for the app, we need to install the requirements listed in the requirements.txt file located in the i18n_scripts directory. This can be done easily by running the following make command:
+```bash
+make translation_requirements
+```
+
+Then, to get the latest translations for all languages use the following command:
+```bash
+make pull_translations
+```
+This command runs [`atlas pull`](https://github.com/openedx/openedx-atlas) to download the latest translations files from the [openedx/openedx-translations](https://github.com/openedx/openedx-translations) repository. These files contain the latest translations for all languages. In the [openedx/openedx-translations](https://github.com/openedx/openedx-translations) repository each language's translations are saved as a single file e.g. `i18n/src/main/res/values-uk/strings.xml` ([example](https://github.com/openedx/openedx-translations/blob/04ccea36b8e6a9889646dfb5a5acb99686fa9ae0/translations/openedx-app-android/i18n/src/main/res/values-uk/strings.xml)). After these are pulled, each language's translation file is split into the App's modules e.g. `auth/src/main/res/values-uk/strings.xml`.
+   
+  After this command is run the application can load the translations by changing the device (or the emulator) language in the settings.
+
+### Using Custom Translations
+
+By default, the command `make pull_translations` runs [`atlas pull`](https://github.com/openedx/openedx-atlas) with no arguments which pulls translations from the [openedx-translations repository](https://github.com/openedx/openedx-translations).
+
+You can use custom translations on your fork of the openedx-translations repository by setting the following configuration parameters:
+
+- `--revision` (default: `"main"`): Branch or git tag to pull translations from.
+- `--repository` (default: `"openedx/openedx-translations"`): GitHub repository slug. There's a feature request to [support GitLab and other providers](https://github.com/openedx/openedx-atlas/issues/20).
+
+Arguments can be passed via the `ATLAS_OPTIONS` environment variable as shown below:
+``` bash
+make ATLAS_OPTIONS='--repository=<your-github-org>/<repository-name> --revision=<branch-name>' pull_translations
+```
+Additional arguments can be passed to `atlas pull`. Refer to the [atlas documentations ](https://github.com/openedx/openedx-atlas) for more information.
+
+### Testing Translations
+
+Until the [pull request #317](https://github.com/openedx/openedx-app-android/pull/317) is merged, translations needs to be pulled from the testing branch `Zeit-Labs/openedx-translations` repo under `fc_55_sample` branch with the following options:
+``` bash
+make ATLAS_OPTIONS='--repository=Zeit-Labs/openedx-translations --revision=fc_55_sample' pull_translations
+```
```

Comment:
How about removing this section from the README?
It seems that the PR description is sufficient for testing